src/GravityFormsExtensions.php

```diff
@@ -101,6 +101,7 @@ private function hooks() {
 		add_filter( 'gform_confirmation_settings_fields', [ $this, 'p4_gf_confirmation_settings' ], 10, 3 );
 		add_filter( 'gform_confirmation', [ $this, 'p4_gf_custom_confirmation' ], 10, 3 );
 		add_filter( 'gform_field_css_class', [ $this, 'p4_gf_custom_field_class' ], 10, 3 );
+		add_filter( 'gform_form_args', [ $this, 'p4_gf_enforce_ajax' ], 10, 3 );
```

Comment:
Just a note for future, the 4th argument in `add_filter()` function is `$accepted_args` so here in this case it should be 1.

As we are working on PHPCS [PR](https://github.com/greenpeace/planet4-master-theme/pull/1880/commits/44ae2a0b6dc1a0b20e4a78fccae1dbe4ee135be7), while rebasing it will get alerted, and we will fix it there.